lib/roo_on_rails/checks/sidekiq/settings.rb

```diff
@@ -0,0 +1,29 @@
+require 'roo_on_rails/checks/base'
+require 'thor'
+
+module RooOnRails
+  module Checks
+    module Sidekiq
+      class Settings < Base
+        requires Sidekiq
+        def intro
+          'Checking Sidekiq settings.yml...'
+        end
+
+        def call
+          fail! 'No sidekiq settings found' unless File.exist?('config/sidekiq.yml')
+          return if File.read('config/sidekiq.yml') == yaml_template
```

Comment:
```ruby
# in your initializer
Sidekiq.configure_server do |x|
  x.options[:concurrency] = ENV.fetch('SIDEKIQ_THREADS')
  x.options[:queues] = %w[realtime ... within1day]
end
```

This could let you get rid of the generated config file and keep your Procfile line down to `bundle exec sidekiq`.